File: Core Java/interface-polymorphism/src/com/techlabs/boy/Boy.java
package com.techlabs.boy;

import com.techlabs.iemotionable.IEMotionable;
import com.techlabs.imannerable.IMAnnerable;

public class Boy implements IEMotionable,IMAnnerable {

	@Override
	public void wish() {
		System.out.println("Boy is wishing.....");
	}

	@Override
	public void depart() {
		System.out.println("Boy is upto depart....");
	}

	@Override
	public void cry() {
		System.out.println("Boy is crying....");
	}

	@Override
	public void laugh() {
		System.out.println("Boy is Laughing....");
	}

}

File: DOTNET/C#/InterfacePolymorphismApp/InterfacePolymorphismApp/Boy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfacePolymorphismApp
{
    public class Boy : IMAnneravle, IEMotionable
    {
        public void Cry()
        {
            Console.WriteLine("Boy is crying....");
        }

        public void Depart()
        {
            Console.WriteLine("Boy is upto depart.....");
        }

        public void Laugh()
        {
            Console.WriteLine("Boy is lauging.....");
        }

        public void Wish()
        {
            Console.WriteLine("Boy is wishing....");
        }
    }
}
